src/Datadog.Trace/Tracer.cs

```diff
@@ -217,10 +217,48 @@ public Span StartSpan(string operationName, ISpanContext parent = null, string s
                 span.SetTag(Tags.Env, env);
             }
 
+            // Apply any global tags
+            if (((TraceContext)traceContext).Tracer.Settings.GlobalTags.Count > 0)
+            {
+                foreach (var entry in ((TraceContext)traceContext).Tracer.Settings.GlobalTags)
```

Comment:
Same as above comment.
```suggestion
                foreach (var entry in Settings.GlobalTags)
```

Reply:
👍